src/haddock/core/typing.py

```diff
@@ -30,6 +30,7 @@
     Generic,
     Iterable,
     Iterator,
+    List,
```

Comment:
`List` is not used

Reply:
Used in `libontology`.
Imported by other scripts.